### PR TITLE
Implemented TokenManagerHook and migrated to Buidler

### PR DESCRIPTION
**Pull request by willjgriff**
Implementing TokenManagerHook allows us to prevent transfer of an associated MiniMeToken when an address also could not redeem.

Migrated to Buidler, including tests.

### CODE REVIEW

File: scripts/buidler-hooks.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    await minime.changeController(tokens.address)
+    log(`> Change minime controller to tokens app`)
+    await tokens.initialize([minime.address, true, 0])
+    log(`> Tokens app installed: ${tokens.address}`)
+  },
+
+  getInitParams: async function({}, bre) {
+    return [
+      tokens.address,
```
Comment (fabriziovigevani) [inline scripts/buidler-hooks.js:35]: Should this be `minime.address` ?

File: test/voting.js [hunk 2]
```diff
@@ -695,18 +710,18 @@ contract('Voting App', ([root, holder1, holder2, holder20, holder29, holder51, n
...
-            assert.isTrue(isOpen, 'vote should be open')
-            assert.isFalse(isExecuted, 'vote should not have been executed')
+            assert.isTrue(open, 'vote should be open')
+            assert.isFalse(executed, 'vote should not have been executed')
         })
 
         it('creating vote as holder2 does not execute vote', async () => {
             const voteId = createdVoteId(await voting.newVote(EMPTY_SCRIPT, 'metadata', true, { from: holder2 }))
-            const [isOpen, isExecuted] = await voting.getVote(voteId)
+          const { open, executed } = await voting.getVote(voteId)
```
Comment (fabriziovigevani) [inline test/voting.js:721]:
```suggestion
            const { open, executed } = await voting.getVote(voteId)
```

File: test/voting.js [hunk 3]
```diff
@@ -695,18 +710,18 @@ contract('Voting App', ([root, holder1, holder2, holder20, holder29, holder51, n
             await voting.vote(voteId, true, { from: holder1 })
             await voting.vote(voteId, true, { from: holder2 })
 
-            const [isOpen, isExecuted] = await voting.getVote(voteId)
+          const { open, executed } = await voting.getVote(voteId)
```
Comment (fabriziovigevani) [inline test/voting.js:713]:
```suggestion
            const { open, executed } = await voting.getVote(voteId)
```

File: scripts/buidler-hooks.js (hunk 1 above)
Comment (willjgriff) [reply scripts/buidler-hooks.js:35]: Yeah probas, I copied it from the Aragon Voting app.

**Pull request merged**